database/migrations/20200916191917_users.js

```diff
@@ -0,0 +1,15 @@
+exports.up = async function (knex) {
+    await knex.schema
+    .createTable("users", (tbl) => {
+        tbl.increments("id").unsigned().primary()
+        tbl.text("username", 20).unique().notNullable()
+        tbl.text("password", 128).notNullable();
```

Comment:
If these length limits are here can we check in the `add()` or `/login` if these constraints are met and send back useful error to the user.